scripts/meson.build

```diff
@@ -1,14 +1,16 @@
-prog_python = import('python').find_installation('python3')
+prog_python = import('python').find_installation('python3', required : false, modules : ['cffi'])
```

Comment:
You could try to use the `required` value from a [feature](https://mesonbuild.com/Build-options.html#features), so that users can enforce this module being present (and fail the configure step).

Reply:
Done, thanks for the tips!